src/test/java/core/basesyntax/service/strategy/HandleTransactionStrategyImplTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.strategy.handlers.BalanceHandler;
+import core.basesyntax.service.strategy.handlers.ReturnHandler;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class HandleTransactionStrategyImplTest {
+
+    private static HandleTransactionStrategyImpl handleTransactionStrategy;
+
+    @BeforeAll
+    static void beforeAll() {
+        handleTransactionStrategy = new HandleTransactionStrategyImpl();
+        Storage.fruitsMap.clear();
+    }
+
+    @Test
+    void get_correctOperation_ok() {
+        FruitTransaction.Operation balanceOperation = FruitTransaction.Operation.BALANCE;
+        FruitTransaction.Operation returnOperation = FruitTransaction.Operation.RETURN;
+        Class expectedBalance = new BalanceHandler().getClass();
```

Comment:
why class??

Reply:
because here I check which handler class is returned by strategy

Reply:
lets use TransactionHandler